frontend/src/components/pages/connect/Connector.Details.tsx

```diff
@@ -125,21 +132,28 @@ const KafkaConnectorMain = observer(
                                 <Tooltip placement="top" isDisabled={canEdit !== true} label={'You don\'t have \'canEditConnectCluster\' permissions for this connect cluster'} hasArrow={true}>
                                     <Button
                                         variant="outline"
-                                        style={{width: '200px'}}
+                                        style={{ width: '200px' }}
                                         disabled={(() => {
```

Comment:
I think this property should be `isDisabled`. There some some TS bug in the UI libraryI think, because it doesn't warn us.

Reply:
good catch